src/main/groovy/com/blackducksoftware/integration/hub/detect/hub/HubManager.groovy

```diff
@@ -173,6 +172,7 @@ class HubManager implements ExitCodeReporter {
         builder.setProjectLevelAdjustments(detectConfiguration.getProjectLevelMatchAdjustments())
         builder.setPhase(detectConfiguration.getProjectVersionPhase())
         builder.setDistribution(detectConfiguration.getProjectVersionDistribution())
+        builder.setProjectTier(detectConfiguration.projectTier)
         ProjectRequest projectRequest = builder.build()
```

Comment:
Let's try for consistency here. If what exists is intolerable, it should all be updated. If what exists is tolerable, it should be emulated.